cfme/utils/appliance/__init__.py

```diff
@@ -1658,7 +1665,11 @@ def wait_for_host_address(self):
 
     @property
     def is_ssh_running(self):
-        return net_check(ports.SSH, self.hostname, force=True)
+        if self.openshift_creds and 'hostname' in self.openshift_creds:
+            hostname = self.openshift_creds['hostname']
```

Comment:
**REQUIRED:** So this verifies that the host hosting the container has ssh open, because we use this for the oc command?

Reply:
yes